scripts/components/CollectionEdit.js

```diff
@@ -21,7 +21,7 @@ export default class CollectionEdit extends Component {
 
   render() {
     const {collection} = this.props;
-    const {label, schema, uiSchema, displayFields, busy} = collection;
+    const {schema={}, uiSchema={}, displayFields=[], label, busy} = collection;
```

Comment:
Me neither 2 days ago :p Now I see so many places this should be used it's embarrassing.